shared/core/types/proto/block.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package core;
+
+option go_package = "github.com/pokt-network/pocket/shared/core/types";
+
+import "google/protobuf/timestamp.proto";
+
+message BlockHeader {
+  uint64 height = 1;
+  string networkId = 2; // used to differentiate what network the chain is on (Tendermint legacy)
```

Comment:
Stupid question: Are we keeping these tendermint legacy fields because we'll want to port these values from V0 when V1 goes live and the protocol upgrades?

Reply:
Nah, all of the state will be ported to the genesis. We will have completely new structures.

It was the sort of thing we copied over "just cause" and I wanted to keep this PR to ONLY consolidation.

Updating comment to this:

```
  // TECHDEBT: Re-evaluate whether we want/need these fields (copied over from Tendermint's structures)
  //           at all. They were added earlier on in the implementation but might not be needed at all
```

Reply:
Gotcha. Thanks for clarifying